spec/stack_master/validator_spec.rb

```diff
@@ -5,16 +5,16 @@
   let(:stack_name) { 'myapp_vpc' }
   let(:stack_definition) do
     StackMaster::StackDefinition.new(
-      region: 'us-east-1',
-      stack_name: stack_name,
-      template: 'myapp_vpc.json',
-      tags: { 'environment' => 'production' },
-      base_dir: File.expand_path('spec/fixtures'),
+        region: 'us-east-1',
+        stack_name: stack_name,
+        template: 'myapp_vpc.json',
+        tags: {'environment' => 'production'},
+        base_dir: File.expand_path('spec/fixtures'),
```

Comment:
Maybe we should implement a linter, because it seems like your IDE is enforcing something different to others.

Reply:
Maybe...